lib/migration/config/v1/executor/container_vm.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
Range is for update files or not? I'm wondering if we could batch update all files.